core/templates/dev/head/pages/exploration_player/FeedbackPopupDirective.js

```diff
@@ -20,7 +20,7 @@
 // follows:
 //
 // <some-html-element popover-placement="bottom"
-//                    uib-popover-template="popover/feedback"
+//                    uib-popover-template="FEEDBACK_POPOVER_HTML"
```

Comment:
Nice catch, but shouldn't it be `uib-popover-template="'feedback_popup_container.html'"`?

Reply:
Sorry, did it by mistake. Corrected now.